src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  let vocationCost = 0;
+
+  if (days >= 7) {
+    vocationCost += days * cost - bigDiscount;
+  } else if (days >= 3) {
```

Comment:
You can remove the 'else' keyword since the 'if' block above contains a 'return' statement. This simplifies the control flow and makes the code easier to read.